challenge.js

```diff
@@ -42,7 +42,7 @@
  */
 
 // ANSWER
-
+// In the case of needing to refer to it from the DOM, would the parent method need to be used twice in order to reach A from E (.parent().parent())?
```

Comment:
The goal here is to write a CSS selector string so we can select E without needing any references to any of its "relatives". Here is an explanation and solution:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.

1/2